src/total-count/dtos/counting.dto.ts

```diff
@@ -0,0 +1,30 @@
+import { IsEnum } from 'class-validator';
+import { Action } from '../enums/action.enum';
+import { Type } from '../enums/type.enum';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class CountingDto {
+  mentorId?: number;
+
+  @ApiProperty({
+    description:
+      'countMentorBoard: 멘토 게시글 조회수, countHelpYouComment: 도와주세요 댓글 조회수, countMentorBoardLike: 멘토 게시글 좋아요 수, countBadge: 멘토 뱃지 수, countReview: 멘토 후기 수',
+    enum: Type,
+    default: Type.CountMentorBoard,
+  })
+  @IsEnum(Type, {
+    message:
+      'type은 countMentorBoard, countHelpYouComment, countMentorBoardLike, countBadge, countReview 중 하나만 가능합니다.',
+  })
+  type: Type;
+
+  @ApiProperty({
+    description: 'increment: 증가, decrement: 감소',
+    enum: Action,
+    default: Action.Increment,
+  })
+  @IsEnum(Action, {
+    message: 'action은 increment 또는 decrement만 가능합니다.',
+  })
+  action: Action;
```

Comment:
swagger문서 상에서는 default값이 있는데 실제 코드에선 없네요
```suggestion
  action: Action = Action.Increment;
```